benchmarks/apache-spark/src/main/scala/org/renaissance/apache/spark/NaiveBayes.scala

```diff
@@ -11,20 +11,19 @@ import org.apache.spark.mllib.classification.NaiveBayesModel
 import org.apache.spark.mllib.linalg.Vectors
 import org.apache.spark.mllib.regression.LabeledPoint
 import org.apache.spark.rdd.RDD
+import org.renaissance.Benchmark
+import org.renaissance.Benchmark._
+import org.renaissance.BenchmarkContext
 import org.renaissance.BenchmarkResult
-import org.renaissance.Config
-import org.renaissance.CompoundResult
+import org.renaissance.BenchmarkResult.Validators
 import org.renaissance.License
-import org.renaissance.RenaissanceBenchmark
-import org.renaissance.SimpleResult
-import org.renaissance.Benchmark._
 
 @Name("naive-bayes")
 @Group("apache-spark")
 @Summary("Runs the multinomial naive Bayes algorithm from the Spark MLlib.")
 @Licenses(Array(License.APACHE2))
 @Repetitions(30)
-class NaiveBayes extends RenaissanceBenchmark with SparkUtil {
+class NaiveBayes extends Benchmark with SparkUtil {
```

Comment:
Shouldn't this one also get its parameters?

Reply:
I think I only added parameters to benchmarks which explicitly checked for "test" run and thus created a need for a different configuration. For other benchmarks, there are certainly candidates for configurable parameters, but some benchmarks have many of them and I could not tell which ones are worth configuring and which ones not.